command/s3_put_test.go

```diff
@@ -323,6 +374,7 @@ func TestS3LocalFilesIncludeFilterPrefix(t *testing.T) {
 			} else {
 				localFilesIncludeFilterPrefix = prefix
 			}
+			//set visibility based on the prefix
```

Comment:
Could you explain what you meant in this comment?

Reply:
I meant to take that out. Thanks!